Doc/howto/clinic.rst

```diff
@@ -162,28 +162,20 @@ For the sake of clarity, here's the terminology we'll use with Argument Clinic:
 Tutorial
 ========
 
-The best way to get a sense of how Argument Clinic works is to
-convert a function to work with it.  Here, then, are the bare
-minimum steps you'd need to follow to convert a function to
-work with Argument Clinic.  Note that for code you plan to
-check in to CPython, you really should take the conversion farther,
-using some of the :ref:`advanced concepts <clinic-howtos>`
-you'll see later on in the document,
-like :ref:`clinic-howto-return-converters`
-and :ref:`clinic-howto-self-converter`.
-But we'll keep it simple for this walkthrough so you can learn.
-
-First, make sure you're working with a freshly updated checkout
-of the CPython trunk.
```

Comment:
I don't think this brings much value in the tutorial. The tutorial is targeted at the core dev and contributor audience; working on an old checkout seems like a very little likely scenario, IMO.

Reply:
While they know, it might be useful as a reminder.  If being a few commits behind is not an issue, then the reminder is not essential and can be removed.

Reply:
I think it'll be hard to align the tutorial according to Diátaxis principles until we've extracted the "how to apply Argument Clinic to CPython" bits from it. Perhaps we should do that first.

What do you think, @ezio-melotti?